awx_collection/plugins/modules/tower_job_template.py

```diff
@@ -12,409 +13,532 @@
                     'status': ['preview'],
                     'supported_by': 'community'}
 
-
 DOCUMENTATION = '''
 ---
 module: tower_job_template
-author: "Wayne Witzel III (@wwitzel3)"
+author: "John Westcott IV (@john-westcott-iv)"
 version_added: "2.3"
-short_description: create, update, or destroy Ansible Tower job template.
+short_description: create, update, or destroy Ansible Tower job templates.
 description:
     - Create, update, or destroy Ansible Tower job templates. See
       U(https://www.ansible.com/tower) for an overview.
 options:
     name:
       description:
-        - Name to use for the job template.
+        - Name of this job template.
+      required: True
+      type: str
+    new_name:
+      description:
+        - Setting this option will change the existing name (looed up via the name field.
       required: True
       type: str
     description:
       description:
-        - Description to use for the job template.
+        - Optional description of this job template.
+      required: False
       type: str
+      default: ''
     job_type:
       description:
         - The job type to use for the job template.
       required: False
-      choices: ["run", "check"]
       type: str
+      default: 'run'
+      choices:
+        - 'run'
+        - 'check'
+        - 'scan'
```

Comment:
Previously, we explicitly got rid of the "scan" option. I thought the API had purged it, but it is also in the AWX collection list of changes.